resources/js/components/ui/Table/TableColumnVisibilityController.tsx

```diff
@@ -24,7 +24,11 @@ export const TableColumnVisibilityController: VoidComponent = () => {
   const api = createMemo(() => popover.connect(state, send, normalizeProps));
   return (
     <div class={s.columnVisibility}>
-      <Button class="pressedWithShadow" {...api().triggerProps} disabled={!table.getAllLeafColumns().length}>
+      <Button
+        class="pressedWithShadow border-input-border"
```

Comment:
kurcze szkoda, że raz jest camelCase a raz kebab-case

Reply:
Dosyć konsekwentnie nasze klasy nazywamy camelCasem, zresztą klasy z module.scss używane są potem jako identyfikatory w TS więc chyba nawet tak trzeba. Przynajmniej od razu widać że to nasza klasa a nie wbudowana.